frontend/src/components/panels/project_filters_panel/filter_map_components/DateRangeSlider.js

```diff
@@ -1,9 +1,73 @@
 import React from "react";
+import ReactDOM from 'react-dom';
+import Tooltip from 'rc-tooltip';
+import Slider, { Range } from 'rc-slider';
+import 'rc-slider/assets/index.css';
+import 'rc-tooltip/assets/bootstrap.css';
 
-const DateRangeSlider = () => (
-    <div className="my-1">
-        <input type="range" className="custom-range" id="customRange1"/>
-    </div>
-);
+const createSliderWithTooltip = Slider.createSliderWithTooltip;
+const Handle = Slider.Handle;
+
+const dotStyle = {
+	'bottom': '-10px',
+    'width': '1px',
+    'border': '1px solid #e9e9e9',
+    'border-radius': '0%',
+    'margin-left': '-1px',
+};
```

Comment:
To avoid camelCasing these style arguments, consider assigning these attributes to a css class and declaring it in the style subfolder (`frontend/src/styles/panels/_filters_panel.scss`)